CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Forge
+
+#### Added
+
+- `--print-gas-usage`, short `-g` flag to print gas usage for each test
+
```

Comment:
Why this is hidden under a flag?

Reply:
I don't think it's useful to always display it

Reply:
Tbh this flag is weird, I would just always print it

Reply:
If we do that default, we will need to update our all e2e tests 😨

Reply:
I don't have a very strong opinion on that. We can always print it if you all think that's useful.